src/pages/rental/rental.ts

```diff
@@ -62,18 +62,63 @@ export class RentalPage {
    * Pushes page on nav to allow users to finish renting the items.
    */
   onContinue() {
-    this.navCtrl.push(RentalDetailsPage);
+    let items;
+    this.items.take(1).subscribe(i => items = i.rentals);
```

Comment:
`i` is for items. I couldn't use the name `items` since it is already used as a local variable and I figured the name of the variable did not matter much since it is simply what is returned by the object in the store and isn't used anywhere else.

I can change though it if you think it is worth and can think of an appropriate name!